src/Controller/WidgetController.php

```diff
@@ -43,6 +44,17 @@ public function generateDiagram()
         return $response;
     }
 
+    /**
+     * @Route("/widget/schema/{type}", name="widgets_schema")
+     */
+    public function generateSchema($type)
+    {
+        $schema = new DiagramSchema($type);
+        echo stripslashes(json_encode($schema->generateSchema(), JSON_PRETTY_PRINT));
+        // $schema->generateSchemaArray();
+        exit();
```

Comment:
We should follow the best practices about how we return JSON data from controllers. Could you follow the same approach we use in DiagramsGeneratorController::generate() ?